frontend/Makefile

```diff
@@ -16,24 +16,36 @@ init:
 dev.end:
 	@make --directory .. dev.backend.end
 
-test.admin-unit:
+dev-build:
+	@lerna run build
+
+dev-test.admin-unit:
 	@lerna run test:unit --scope admin-ui
 
-test.consumer-unit:
+dev-test.consumer-unit:
```

Comment:
We need to distinguish between "dev" and "ci" tests somehow. The "dev" tests run the frontends without using docker and the "ci" tests use docker. Docker is necessary on the ci to make all the npm packages available in a clean way. On a dev machine, it's slower to run the tests that way and I find it a bit less practical.